generators/openapi-client/files.js

```diff
@@ -100,6 +100,8 @@ function writeFiles() {
                 return;
             }
 
+            const openApiToolsDependencyVersion = '0.2.1';
```

Comment:
I think we should put the version in this file: https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-constants.js

Reply:
then in pom.xml and gradle.properties, use this constant